tests/res/fluka/mock/generate.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+"""
+This script collects all fluka output files in the current directory and generates a bash script named rfluka.
+
+Generated bash script will contain all fluka output files as base64 encoded strings.
+Running generated script will create all fluka output files in the current directory.
+"""
+
+import base64
+import os
+from pathlib import Path
+
+__FILE_NAME_AND_CONTENT_TEMPLATE = """
+FILE_NAME_{index}="{file_name}"
+FILE_NAME_CONTENT_{index}="{file_content}"
+sleep 1
+printf '%s' "$FILE_NAME_CONTENT_{index}" | base64 -d -i > $FILE_NAME_{index}
+echo "Generated $FILE_NAME_{index}"
+"""
+
+__OUTPUT_SCRIPT_NAME = "rfluka"
+
+
+def to_bash_lines(index: int, file_name: str, file_content: bytes) -> str:
```

Comment:
Maybe this should be common between Fluka and SH12A mocks. These two issues could be solved in PR

Reply:
Sure.

Should all mocks and their dependencies be placed in a single directory then?

Something like this:

- tests/res/mocks/lib/bash.py
- tests/res/mocks/generate_rfluka.py
- tests/res/mocks/generate_shieldhit.py

Reply:
Good idea